app/geo/gui/src/ts/twxplore/gui/geo/components/navbar/Navbar.tsx

```diff
@@ -1,31 +1,56 @@
-import {ActiveNavbarItem} from 'twxplore/gui/geo/components/navbar/ActiveNavbarItem';
-import {Hrefs} from 'twxplore/gui/geo/Hrefs';
-import * as React from 'react';
-import {Link, RouteComponentProps, withRouter} from 'react-router-dom';
-import {Nav, Navbar as BootstrapNavbar, NavbarBrand, NavItem, NavLink,} from 'reactstrap';
+import {ActiveNavbarItem} from "twxplore/gui/geo/components/navbar/ActiveNavbarItem";
+import {Hrefs} from "twxplore/gui/geo/Hrefs";
+import * as React from "react";
+import {Link, RouteComponentProps, withRouter} from "react-router-dom";
+import {
+  Nav,
+  Navbar as BootstrapNavbar,
+  NavbarBrand,
+  NavItem,
+  NavLink,
+} from "reactstrap";
 
 interface Props extends RouteComponentProps {
-    activeNavItem?: ActiveNavbarItem;
+  activeNavItem?: ActiveNavbarItem;
 }
 
 const Navbar: React.FunctionComponent<Props> = ({activeNavItem}) => {
-    return (
-        <div>
-            <BootstrapNavbar className="py-0" color="light" light expand="md">
-                <NavbarBrand href={Hrefs.home}>twxplore</NavbarBrand>
-                <Nav className="pb-2 pr-4">
-                    <NavItem active={activeNavItem === ActiveNavbarItem.Home}>
-                        <NavLink
-                            active={activeNavItem === ActiveNavbarItem.Home}
-                            tag={Link}
-                            to={Hrefs.home}
-                        >
-                            Home
-                        </NavLink>
-                    </NavItem>
-                </Nav>
-            </BootstrapNavbar>
-        </div>);
-}
+  return (
+    <div>
+      <BootstrapNavbar className="py-0" color="light" light expand="md">
```

Comment:
Didn't realize we still had Bootstrap stuff hanging around. This needs to be switched to Material, and the reactstrap dependency eliminated.